src/main/java/athleticli/ui/Parser.java

```diff
@@ -282,11 +295,12 @@ public static AddSleepCommand parseSleepAdd(String commandArgs) throws AthletiEx
         if (startMarkerPos > endMarkerPos) {
             throw new AthletiException("Please specify the start time of your sleep before the end time.");
         }
-    
-        String startTime = commandArgs.substring(startMarkerPos + startMarkerConstant.length(), endMarkerPos).trim();
+
+        String startTime =
+                commandArgs.substring(startMarkerPos + startMarkerConstant.length(), endMarkerPos).trim();
         String endTime = commandArgs.substring(endMarkerPos + endMarkerConstant.length()).trim();
 
-        if(startTime.isEmpty() || endTime.isEmpty()) {
+        if (startTime.isEmpty() || endTime.isEmpty()) {
             throw new AthletiException("Please specify both the start and end time of your sleep.");
```

Comment:
Some AthletiExceptions can still be factored out as string constants in messages class